Intersect.Server/Entities/Entity.cs

```diff
@@ -1521,8 +1640,8 @@ byte projectileDir
             if (spellBase.HitAnimationId != Guid.Empty &&
                 (spellBase.Combat.Effect != StatusTypes.OnHit || onHitTrigger))
             {
-                deadAnimations.Add(new KeyValuePair<Guid, sbyte>(spellBase.HitAnimationId, (sbyte) Directions.Up));
-                aliveAnimations.Add(new KeyValuePair<Guid, sbyte>(spellBase.HitAnimationId, (sbyte) Directions.Up));
+                deadAnimations.Add(new KeyValuePair<Guid, sbyte>(spellBase.HitAnimationId, (sbyte) Direction.Up));
+                aliveAnimations.Add(new KeyValuePair<Guid, sbyte>(spellBase.HitAnimationId, (sbyte) Direction.Up));
```

Comment:
The same "why is this an sbyte instead of direction" question

Reply:
Is there something else stored in the values besides directions?

Reply:
![imagen](https://user-images.githubusercontent.com/17498701/222857566-70d18f80-774f-4807-8bb0-01f489dcca7c.png)

Reply:
![imagen](https://user-images.githubusercontent.com/17498701/222857605-4c1d2cd8-42db-42a6-a73c-e0854bec2456.png)